issue_template.md

```diff
@@ -0,0 +1,29 @@
+### **Description**
+
+Please describe the issue you are facing or a feature you would like to be added to the SDK.
+
+<!-- If making a feature request, remove the below information -->
+### **Repro Steps**
+
+Please list the steps used to reproduce your issue.
+
+1.
+2.
+
+### **Details**
+
+1. Which SDK version are you using?
+    - e.g. 1.5.0
+2. Which OS version did you experience the issue on?
+    - e.g. iOS 11, Android 8.1.0
```

Comment:
Can you remove the "Android 8.1.0" text from here? Rather have an example for macOS instead